modules/virt-deleting-templates-from-custom-namespace.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-deleting-templates-from-custom-namespace.adoc
+
+[id="virt-deleting-templates-from-custom-namespace_{context}"]
+= Deleting templates from a custom namespace
```

Comment:
Perhaps the behavior of what happens when a template is modified or deleted can be moved to the beginning, and using a custom namespace will allow them to be modified/deleted from default namespace as required by the admin

Reply:
Hmmm - currently the logic is to add the custom namespace with the templates, and then be able to delete them. Would prefer to leave the sequence of topics in this order.

Reply:
Adding or creating an object precedes deleting or modifying an object. Therefore, let's retain the current logical sequence.